### PR TITLE
Integrate legal with member relations stages

### PR DESCRIPTION
**Pull request by clausmullie**
@ElenaFdR, when @MirjamvT and I were reviewing the legal document we agreed it was extremely similar to member relations index.

I have made a draft pull request here to see what merging them would look like. I added all the content and think it works. Slight doubt about the 'dispute assumptions' list as it sounds scary - but I guess that's a good thing for us to continue working on.

### CODE REVIEW

File: activities/member-relations/index.md [hunk 1]
```diff
@@ -8,31 +8,58 @@ This outlines the stages by which a public organization becomes a founding membe
...
 
-From 2020, each member will have an annual membership contract.
+A mediation structure will be set up to handle any disputes that may arise between members and the Foundation or between individual members.
 
-With the annual membership contract, members become co-owners of the Foundation. This includes steering and setting priorities for the Foundation and voting in the General Assembly. 
+### Assumptions
+* This contract will be governed by Dutch law.
+* Members will not pay dues through procurement or tendering procedures.
+* The procedures members use to pay their dues will differ based on local legal context -  for example, some could use a grant.
+* Members will not incur obligations to the Foundation beyond paying dues and any other obligations stipulated in the contract.
```
Comment (ElenaFdR) [inline activities/member-relations/index.md]:
```suggestion
* Members will not incur obligations to the Foundation beyond paying dues.
```
This is much more helpful if we can provide a definitive list. "Members do not incur any obligations beyond the obligations in the contract' seems more weasely (though that is more futureproofed).

Comment (clausmullie) [reply activities/member-relations/index.md]: True - I guess we can always update this once we get the contract defined

**Comment by ElenaFdR** [comment]
This is still missing some hyperlinks (for example, to the bylaws, governance and financial model).

**Comment by clausmullie** [comment]
only bylaws remaining now, but while reading I thought they should probably be called the 'association' bylaws no?

**Comment by MirjamvT** [comment]
@clausmullie, i think 'association bylaws' is good.

**Comment by bvhme** [comment]
I was wondering what the relation is between 'statuten', 'artikelen van oprichting' en 'regelement' in dutch, and which ones are the bylaws...

File: activities/member-relations/index.md [hunk 2]
```diff
@@ -8,13 +8,18 @@ This outlines the stages by which a public organization becomes a founding membe
 
 ## 1. Expression of interest
 
-At this stage, the public organization confirms their interest in becoming a member by signing the '[Expression of Interest](expression-of-interest.md)'.
+At this stage, the public organization confirms their interest in becoming a member by signing the '[Expression of Interest](expression-of-interest.md)'.
```
Comment (bvhme) [inline activities/member-relations/index.md]: The link needs no caps and no quotes.

```suggestion
At this stage, the public organization confirms their interest in becoming a member by signing the [expression of interest](expression-of-interest.md).
```

File: activities/member-relations/index.md [hunk 3]
```diff
@@ -8,13 +8,18 @@ This outlines the stages by which a public organization becomes a founding membe
...
-At this stage, the public organization confirms their interest in becoming a member by signing the '[Expression of Interest](expression-of-interest.md)'.
+At this stage, the public organization confirms their interest in becoming a member by signing the '[Expression of Interest](expression-of-interest.md)'. 
+
+This grants both parties the right to discuss this collaboration publicly. It's signed by a policy lead and/or technical development lead in the public organization.
+
+This stage is over when the expression of interest has been signed.
 
 ## 2. Founding membership proposal
 
 At this stage, the Foundation and the public organization work together on the '[Founding Membership Proposal](founding-membership-proposal.md)'.
```
Comment (bvhme) [inline activities/member-relations/index.md]:
```suggestion
At this stage, the Foundation and the public organization work together on the [founding membership proposal](founding-membership-proposal.md).
```

File: activities/member-relations/index.md [hunk 4]
```diff
@@ -25,14 +30,36 @@ This stage is over when both parties agree to the proposal.
...
+From 2020, each member has a regular membership contract.
+
+With regular membership, members become co-owners of the Foundation. This means members can vote in the general assembly and are responsible for and steer the Foundation’s operations, including priorities and annual budget for the organization. [Read more about Foundation governance](../../organization/governance-model.md).
+
+All members pay membership yearly dues. [See financial model](../../organization/financial-model.md). Failure to pay can result in termination of membership as stipulated in the foundation bylaws. 
+
+A mediation structure will be set up to handle any disputes that may arise between members and the Foundation or between individual members.
 
-With a regular membership contract, members become co-owners of the Foundation. This includes steering and setting priorities for the Foundation and voting in the general assembly.
+### Assumptions
```
Comment (bvhme) [inline activities/member-relations/index.md]: Needs whitespace after heading.

File: activities/member-relations/index.md [hunk 5]
```diff
@@ -25,14 +30,36 @@ This stage is over when both parties agree to the proposal.
...
+* This contract will be governed by Dutch law.
+* Members will not pay dues through procurement or tendering procedures.
+* The procedures members use to pay their dues will differ based on local legal context -  for example, some could use a grant.
+* Members will not incur obligations to the Foundation beyond paying dues.
+* The Foundation will not incur obligations to individual members by accepting membership dues, other than the governance process set out in the assocation bylaws.
+* Disputes about Foundation activites could include, for example:
```
Comment (bvhme) [inline activities/member-relations/index.md]: This can be a new line of text followed by a list, no need for a sub-list

**Comment by ElenaFdR** [comment]
@bvhme [The Dutch gov't calls it](https://business.gov.nl/starting-your-business/choosing-a-business-structure/association/) a 'deed' with 'statutes'. They also note there are 'house rules' to govern the day to day affairs.

**Comment by ElenaFdR** [comment]
@bvhme Here's a definition of bylaws that matches how we use it: https://en.wikipedia.org/wiki/Bylaw#Organizational_by-laws

**Pull request merged**